pyproject.toml

```
@@ -1,13 +1,82 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools-rust"]
+requires = ["maturin>=0.15,<0.16"]
+build-backend = "maturin"
 
-[tool.isort]
-profile = "black"
+[project]
+name = "dask_sql"
+description = "SQL query layer for Dask"
+maintainers = [{name = "Nils Braun", email = "nilslennartbraun@gmail.com"}]
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
```

Since this is MIT (and not BSD)

```suggestion
    "License :: OSI Approved :: MIT License",
```